It seems that enabling it will cause memory leak on Termux.

Fixes https://github.com/termux/termux-packages/issues/29198

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -570,7 +570,7 @@ cmake_dependent_option(USE_LLVM_BOLT "Optimize libraries with LLVM BOLT."
 # Enable third party mimalloc library to improve memory allocation performance
 # on Windows and AArch64.
 set(USE_MIMALLOC_DEFAULT OFF)
-if(WIN32 OR (CPU_AARCH64 AND NOT APPLE))
+if(FALSE)
   set(USE_MIMALLOC_DEFAULT ON)
 endif()
 option(USE_MIMALLOC "Use mimalloc" "${USE_MIMALLOC_DEFAULT}")
